str_analysis/convert_gangstr_spec_to_expansion_hunter_variant_catalog.py

```diff
@@ -0,0 +1,80 @@
+"""This script converts a GangSTR repeat spec to an ExpansionHunter variant catalog. This simplifies the process of
+switching from GangSTR to ExpansionHunter to genotype a set of loci previously genotyped using GangSTR.
+"""
+
+import argparse
+import collections
+import gzip
+import json
+from pprint import pformat
+import re
+import tqdm
+
+
+def main():
+    p = argparse.ArgumentParser()
+    p.add_argument("-o", "--output-file", help="json file output path")
+    p.add_argument("-v", "--verbose", action="store_true")
+    p.add_argument("gangstr_spec", help="path of the GangSTR repeat spec .bed file")
+    args = p.parse_args()
+
+    if not args.output_file:
```

Comment:
many of the other scripts have file existence checks for input args -- would it be helpful to add one here to check that `args.gangster_spec` exists?